versioned_docs/version-latest/learn/monitoring.md

```diff
@@ -0,0 +1,723 @@
+---
+title: Monitoring
+sidebar_position: 4
+description: Monitoring server with Grafana and Prometheus. Scraping metrics from Subspace Node.
+keywords:
+  - Monitoring
+  - Grafana
+  - Prometheus
+  - Exporter
+  - Metrics
+  - Docker
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Link from '@docusaurus/Link';
+import styles from '@site/src/pages/index.module.css';
+
+
+-----------
+
+### Setting Up Monitoring Infrastructure
+
+**Infrastructure Setup:**
+
+Commence by installing and configuring exporters, such as Node Exporter, on the servers earmarked for monitoring. Occasionally, Prometheus client libraries are integrated into application code to reveal metrics at `<host>/metrics`, facilitating the monitoring of specific applications (e.g., Subspace node leverages this functionality). No alterations are necessary in the exporter configuration files on these servers; the default configurations suffice.
```

Comment:
```suggestion
Commence by installing and configuring metrics exporters, such as Node Exporter, on the servers earmarked for monitoring. Occasionally, Prometheus compatible endpoints are integrated into application code to reveal metrics at `<host>/metrics`, facilitating the monitoring of specific applications (e.g. Subspace node leverages this functionality). No alterations are necessary in the exporter configuration files on these servers; the default configurations suffice.
```

Reply:
Isn't that correct, Jim, `Prometheus client libraries are integrated into application code to reveal metrics at `<host>/metrics` ?

Reply:
improved the readability of this whole list in the Infrastructure Setup Overview section

Reply:
> Isn't that correct, Jim, `Prometheus client libraries are integrated into application code to reveal metrics at `/metrics` ?

They are not always exposed at `/metrics` and it would be the server libraries rather than client libraries?